dotnet/src/SemanticKernel.Core/Functions/KernelFunctionFactory.cs

```diff
@@ -35,6 +35,18 @@ public static class KernelFunctionFactory
         ILoggerFactory? loggerFactory = null) =>
```

Comment:
Not for this PR, but do we want to mark this old method as obsolete (at some time in the future) to encourage developers to use the new method?

Reply:
Agreed, I also think we should obsolete the existing ones, and makes sense to do it in a separate PR.